Azure/sonic-buildimage
# Change the default hw watchdog timer value to 3 mins from 10 mins.

**- What I did**
Reduced the default watchdog timeout to 3 mins.
**- How I did it**

**- How to verify it**
Run /usr/bin/watchdog -e to arm the hw watchdog
Wait for 3 minutes and see if the device reboots
**- Description for the changelog**

During the fast-reboot, the device gets stuck in bad state. 
To avoid the device hang status, this change will be part of enabling watchdog during fast-reboot for MLNX platform.
**- A picture of a cute animal (not mandatory but encouraged)**
